pkg/event/target/mqtt.go

```diff
@@ -110,7 +110,7 @@ func (target *MQTTTarget) send(eventData event.Event) error {
 	}
 
 	token := target.client.Publish(target.args.Topic, target.args.QoS, false, string(data))
-	token.Wait()
+	token.WaitTimeout(reconnectInterval * time.Second)
```

Comment:
`token.Error()` is filled only if `token.WaitTimeout` returned `true`. See [here](https://godoc.org/github.com/eclipse/paho.mqtt.golang#ConnectToken.WaitTimeout).